src/api.js

```diff
@@ -11,3 +11,7 @@ export const getArticleInfo = () => {
 export const getArticle = (key) => {
   return newsApi.get(`/articles/${key}`);
 };
+
+export const getArticleComments = (key) => {
+  return newsApi.get(`/articles/${key}/comments`);
```

Comment:
I think perhaps you should be finishing the promise here and _then_ returning the data formatted, rather than returning the promise.